compiler-rt/lib/builtins/aarch64/emupac.c

```diff
@@ -0,0 +1,115 @@
+#include <stdint.h>
```

Comment:
This new file needs a license header, I guess?

Reply:
Done